src/bin/main.rs

```diff
@@ -355,34 +334,31 @@ fn create_project(file: &str, style: &str) -> Result<(), Box<dyn Error>> {
         ))));
     }
 
-    let mut config = AnyConfig::default();
+    /*let mut config = AnyConfig::default();
     config.paths = Some(Paths {
         code: Some("code/".to_string()),
         docs: Some("docs/".to_string()),
         files: Some(vec![file_name]),
-    });
+    });*/
 
-    let template = match style {
+    let (template, toml) = match style {
         "md" => {
-            config.md = Some(MdParser::default());
-            templates::MD
+            //config.md = Some(MdParser::default());
+            (templates::MD, templates::MD_CONFIG)
         }
         "tex" => {
-            config.tex = Some(TexParser::default());
-            templates::TEX
+            //config.tex = Some(TexParser::default());
+            (templates::TEX, templates::TEX_CONFIG)
         }
         "html" => {
-            config.html = Some(HtmlParser::default());
-            templates::HTML
-        }
-        "bird" => {
-            config.bird = Some(BirdParser::default());
-            templates::BIRD
+            //config.html = Some(HtmlParser::default());
+            (templates::HTML, templates::HTML_CONFIG)
         }
-        _ => "",
+        _ => ("", ""),
     };
 
-    let toml = toml::to_string(&config).unwrap();
+    //let toml = toml::to_string(&config).unwrap();
```

Comment:
Not a fan of commenting out code: If it is gone, it should be gone. It can always be retrieved using version control, in the current revision it is just noise.